locale/pl/foreman.po

```diff
@@ -11986,11 +11987,7 @@ msgid ""
 "* global\n"
 "* configuration\n"
 "* build\n"
-msgstr ""
-"Typem statusu może być:\n"
-"* globalny\n"
-"* konfigurowany\n"
-"* zbudowany\n"
+msgstr "Typem statusu może być:\\n* globalny\\n* konfigurowany\\n* zbudowany\\n"
```

Comment:
This is the line that breaks it. I've fixed it up in Transifex so I expect it to work if you pull again.

Reply:
ok pulling now